docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,319 @@
+# Migrate from Pipedream v1
+
+The newest version of the Pipedream workflow builder introduces features like creating code steps in [Python](/code/python), [Bash](/code/bash) and [Go](/code/go).
+
+It also brings the [Node.js](/code/nodes) steps closer to our [Component API](/components/) which powers sources & actions in the platform.
+We have re-imagined the UX from the ground up, made the product much easier to use and have improved performance. In addition, we are introducing powerful new features including:
+
+* **Edit & test** your workflows in separate editing mode without impacting live workflows
```

Comment:
"Edit and test" -> ampersands should only be used in formal titles, etc. (Dreyer's style guide)